tensorflow/compiler/mlir/tosa/transforms/legalize_utils.cc

```diff
@@ -313,13 +313,9 @@ Value getTosaConst16bitTable(PatternRewriter& rewriter, Operation* op,
   table.push_back(
       static_cast<int16_t>(std::min(std::max(max_val, -32768), 32767)));
 
-  auto element_qtype =
-      UniformQuantizedType::get(true, rewriter.getIntegerType(16),
-                                rewriter.getF32Type(), 1.0f, 0, -32768, 32767);
-  auto const_type = tensorflow::GetTypeFromTFTensorShape({513}, element_qtype);
-  auto storage_type = tensorflow::GetTypeFromTFTensorShape(
-      {513}, element_qtype.getStorageType());
-  auto const_attr = DenseElementsAttr::get(storage_type, llvm::ArrayRef(table));
+  auto const_type = tensorflow::GetTypeFromTFTensorShape({513}, rewriter.getIntegerType(16));
```

Comment:
Why was this change needed?

Reply:
If everything still works for this I'm happy, I don't have context as to why this was like this originally.